web/src/compose_fade.js

```diff
@@ -56,6 +56,10 @@ function change_fade_state($elt, should_fade_group) {
 }
 
 function fade_messages() {
+    if (message_lists.current === undefined) {
+        return;
+    }
```

Comment:
The `setTimeout` block below probably needs this too.

Reply:
Moved the `const $all_groups = message_lists.current.view.$list.find(".recipient_row");` below the check to account for `message_list.current` being not the same above the `expected_msg_list` check in `setTimeout`.